imap_processing/tests/cdf/test_utils.py

```diff
@@ -1,19 +1,37 @@
+"""Tests for the ``cdf.utils`` module."""
+
+from pathlib import Path
+
 import imap_data_access
 import numpy as np
 import xarray as xr
 
-from imap_processing import launch_time
+from imap_processing import imap_module_directory, launch_time
 from imap_processing.cdf.global_attrs import ConstantCoordinates
-from imap_processing.cdf.utils import calc_start_time, write_cdf
+from imap_processing.cdf.utils import calc_start_time, load_cdf, write_cdf
 from imap_processing.swe.swe_cdf_attrs import swe_l1a_global_attrs
 
 
 def test_calc_start_time():
+    """Tests the ``calc_start_time`` function"""
+
     assert calc_start_time(0) == launch_time
     assert calc_start_time(1) == launch_time + np.timedelta64(1, "s")
 
 
+def test_load_cdf():
+    """Tests the ``load_cdf`` function."""
+
+    file_path = Path(
+        f"{imap_module_directory}/tests/cdf/data/imap_codice_l1a_hskp_20100101_v001.cdf"
+    )
+    dataset = load_cdf(file_path)
+    assert isinstance(dataset, xr.core.dataset.Dataset)
```

Comment:
Could you instead write out the cdf, then load that one back from the temporary space and compare them equal? Basically, extend the test below or use a fixture for the initial dataset creation to then re-read that in and compare? This would avoid the extra data file then.